cdap-unit-test/src/test/java/io/cdap/cdap/test/app/TestFrameworkTestRun.java

```diff
@@ -1995,8 +1994,6 @@ public void testMetadataAccessInService() throws Exception {
     Assert.assertTrue(scopeMetadata.get(MetadataScope.SYSTEM).getProperties().containsKey("entity-name"));
     Assert.assertEquals(AppWithMetadataPrograms.METADATA_SERVICE_DATASET,
                         scopeMetadata.get(MetadataScope.SYSTEM).getProperties().get("entity-name"));
-    Assert.assertTrue(scopeMetadata.get(MetadataScope.SYSTEM).getTags()
```

Comment:
Only `explore` tag is removed. Presence of `batch` tag can still be asserted?

Reply:
Added the assertion for "batch"